CompositionRoot/CompositionRoot.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <AssemblyName>Messerli.CompositionRoot</AssemblyName>
+        <RootNamespace>Messerli.CompositionRoot</RootNamespace>
+    </PropertyGroup>
+    <PropertyGroup>
```

Comment:
Add nuget specs

Reply:
Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/2/commits/baf96cbed727b79daafca4a394894fe55c928ba2
Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/2/commits/07c04859232b51fd2e61bc7d4d84b8c726924835